gallery/src/guidelines/FontSize/FontSizePage.tsx

```diff
@@ -0,0 +1,315 @@
+          which won't be discussed here. A block elements can form a container for line boxes, the block's width
+          determining the length of each line and thus, along with the text content, the number of lines. A block
+          element's content area contains not only the content areas of its inline children but also any
+          additional spacing applied to their line boxes. Notice the different in the height of the colored
+          backgrounds below:
+        </p>
+
+        <BlockLayoutExample/>
+        <CodeExample content={BlockLayoutExampleCode}/>
+        <CodeExample language="scss" content={BlockLayoutExampleStyles}/>
+      </section>
+    </section>
+
+    <section>
+      <h3>Additional Nuances</h3>
+      <p>Here is an incomplete list of additional nuances to consider when dealing with inline formatting</p>
+      <ul>
+        <li>
+          Inline elements do not respond to CSS width, height, margin-top, or margin-bottom properties. Their
+          size and vertical spacing only follows the <code className="nx-code">font-size</code>,
+          {' '}<code className="nx-code">line-height</code>, and font metrics as described above, as well as padding
+          and borders.
+        </li>
+        <li>inline-block elements <em>are</em> sizable using the CSS width and height properties</li>
+        <li>
+          inline-block elements use the bottom of the element as their baseline, even if they only contain text
+        </li>
+        <li>
+          Things get more complicated when a line box includes inline elements of multiple heights. In this
+          scenario, the elements are sized individually,
+          including <code className="nx-code">line-height</code> spacing, lined up according to
+          the <code className="nx-code">vertical-align</code> property (by default, so their so their baselines
+          are aligned), and then the line box goes from the highest top to the lowest bottom.
+        </li>
+        <li>
+          Parent elements contribute a zero-width <q>strut</q> character to the line spacing. In other words,
+          if an inline element is in a line box along with a parent element that has a larger line height,
+          the parent's line height (and baseline) will come into play even if the parent has no text of its own
+        </li>
+        <li>
+          Due to the off-center nature of the baseline, vertical-alignment of elements of different font sizes is
+          extremely, unavoidably counterintuitive. See the examples in the vertical alignment section
+          of <a href={firstReferenceUrl}>[1]</a>
+        </li>
+        <li>
+          <p>
+            It is possible to specify a <code className="nx-code">line-height</code> smaller than an element's content
+            height, in which case the leading with be negative and text node content areas will have a height exceeding
```

Comment:
> in which case the leading with be negative and text node content areas will have a height exceeding

Did you mean "in which case the leading *will* be negative and text node content areas will have a height exceeding"?

Reply:
> Did you mean "...in which case the leading will be negative and text node content areas will have a height exceeding..."?
```suggestion
            height, in which case the leading will be negative and text node content areas will have a height exceeding
```